test/utils/fixtures/fixtures.json

```diff
@@ -532,6 +532,20 @@
                     "name": "Edit custom theme settings",
                     "action_type": "edit",
                     "object_type": "custom_theme_setting"
+                },
+                {
+                    "name": "Browse newsletters",
+                    "action_type": "browse",
+                    "object_type": "newsletter"
+                },{
```

Comment:
An unbelievably tiny nit but can we put the braces on separate lines? 😛

Reply:
good catch 😉